tests/testthat/test-carehomes-pmcmc.R

```diff
@@ -157,6 +157,32 @@ test_that("can combine rt calculations over trajectories", {
   expect_equal(res$Rt_all, cmp$Rt_all)
 })
 
+
+test_that("can combine rt calculations over trajectories without reordering", {
```

Comment:
I think that the test missing here is that you can rank the trajectories and have the same trajectories out the end. So say you have trajectories {x1, x2, x3, x4} and you reorder them {x1, x3, x4, x2} then you'd check that `expect_equal(ordered[c(1, 4, 3, 2)], unordered)` say

Reply:
though given you're basically doing the opposite the test is easier here , but I think a more explicit test is worthwhile

Reply:
hopefully the new test "when combining rt calculations the output has the expected order" addresses that